Chat/ClientApp/src/components/LeftPane.tsx

```diff
@@ -3,7 +3,13 @@ import { DefaultButton, Icon, IconButton, Pivot, PivotItem, Stack, StackItem } f
 
 import { leftPaneStyle, paneButtonStyle } from './styles/ChatScreen.styles';
 
-export default (props: any): JSX.Element => {
+export interface LeftPaneProps {
+  onTabClicked(): void;
+}
+
+export default (props: LeftPaneProps): JSX.Element => {
+  const { onTabClicked } = props;
+
   return (
```

Comment:
Have you pulled in latest main? There was 2 buttons here I believe :)

Reply:
Yes there are still buttons there for `Home` and `CalendarAgenda`